source/_integrations/melcloud.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: MELCloud
+description: MELCloud integration
+logo: home-assistant.png
+ha_category:
+  - Climate
+ha_release: 0.106
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@vilppuvuorinen'
+---
+
+The `melcloud` integration integrates Mitsubishi Electric's MELCloud enabled devices into Home Assistant.
+
+## Device support
+
+- Air-to-Air heat pumps, i.e. AC units - **Supported**
```

Comment:
do you mean "e.g." instead of "i.e." here?

Reply:
Not really, but it does look a lot better with "e.g.". Switched.